scripts/build.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
```

Comment:
why arent we use react rewired?

Reply:
@sirpy I took master's build process, react rewired isn't on master. that included to run this script that I had to add + update the webpack config